miso-web/src/main/webapp/scripts/warning_partition.js

```diff
@@ -34,10 +34,15 @@ WarningTarget.partition = {
     if (type === 'display') {
       var warnings = [];
       warnings = Warning.addWarnings([
+        [data.prioritySubprojectAliases && data.prioritySubprojectAliases.length > 0, 'PRIORITY' 
+          + (data.prioritySubprojectAliases.length == 1 ? ' (' + data.prioritySubprojectAliases[0] + ')' : '')],
```

Comment:
Include "(multiple)" instead of nothing for multiple subprojects. Same on other pages

Reply:
I suspect the lab will prefer to know exactly which subprojects are in the pool; maybe it could instead be something like ` 'PRIORITY (' + data.prioritySubprojectAliases.reduce((str, acc) => acc.includes(str) ? acc : (acc + ', ' + str)) + ')' `

Reply:
I think keeping the table warning column smaller is a better idea. They can always go to the pool page and see the warnings per dilution, which would include the subproject